CC/Sounder/recorder_worker.cc

```diff
@@ -425,6 +425,16 @@ herr_t RecorderWorker::initHDF5()
             write_attribute(
                 mainGroup, "CL_TX_GAIN_B", this->cfg_->cl_txgain_vec().at(1));
 
+            // Number of frames for UL data recorded in bit source files
+            write_attribute(mainGroup, "UL_DATA_FRAME_NUM",
+                (int)this->cfg_->ul_data_frame_num());
```

Comment:
```suggestion
                static_cast<int>(this->cfg_->ul_data_frame_num()));
```

Reply:
I removed this cast along with some other ones. We can just write these attributes as unsigned.